x-pack/plugins/upgrade_assistant/public/application/app_context.tsx

```diff
@@ -23,6 +24,8 @@ export interface ContextValue {
   notifications: NotificationsStart;
   isReadOnlyMode: boolean;
   api: ApiService;
+  breadcrumbs: BreadcrumbService;
+  getUrlForApp: CoreStart['application']['getUrlForApp'];
```

Comment:
Nice! I didn't now about the `getUrlForApp` helper. We probably have hardcoded navigation to others apps in many places elsewhere....